src/main/java/io/stargate/sgv2/jsonapi/api/v1/KeyspaceResource.java

```diff
@@ -55,13 +56,20 @@ public class KeyspaceResource {
   public static final String BASE_PATH = "/v1/{keyspace}";
   private final MeteredCommandProcessor meteredCommandProcessor;
 
-  @Inject private DataApiRequestInfo dataApiRequestInfo;
+  private final FeaturesConfig apiFeatureConfig;
+
+  private final OperationsConfig operationsConfig;
 
-  @Inject FeaturesConfig apiFeatureConfig;
+  @Inject private DataApiRequestInfo dataApiRequestInfo;
 
   @Inject
-  public KeyspaceResource(MeteredCommandProcessor meteredCommandProcessor) {
+  public KeyspaceResource(
+      MeteredCommandProcessor meteredCommandProcessor,
+      FeaturesConfig apiFeatureConfig,
+      OperationsConfig operationsConfig) {
     this.meteredCommandProcessor = meteredCommandProcessor;
+    this.apiFeatureConfig = apiFeatureConfig;
```

Comment:
nit: question, the other resources using @Inject on the properties not the resources, is it a big change to make them all work the same ?